reference/format/acl.rst

```diff
@@ -0,0 +1,124 @@
+Access Control Lists
+====================
+
++-----------+-----------------+
+| Filename  | ``acl.yml``     |
++-----------+-----------------+
+| Root Node | none            |
++-----------+-----------------+
+| Options   | * `bindings`_   |
+|           | * `class`_      |
+|           | * `group_name`_ |
+|           | * `label`_      |
+|           | * `permission`_ |
+|           | * `type`_       |
++-----------+-----------------+
+
+The ``acl.yml`` file of a bundle can contain a map with access control lists. Each key of the map
+is the unique name of an ACL while the values for each ACL is a map of options:
+
+.. code-block:: yaml
+    :linenos:
+
+    # src/Acme/DemoBundle/Resources/config/acl.rst
+
+    # an ACL used to protect an action
+    password_management:
+        label: Password Management
+        type: action
+        group_name: ""
+
+    # an ACL used to grant removal permissions to users
+    user_delete:
+        label: Delete Users
+        type: entity
+        class: Acme\DemoBundle\Entity\User
+        permission: DELETE
+
+``bindings``
+------------
+
+**type**: ``map``
+
+When a controller action should be protected by an ACL, its method must be bound to the ACL. The
+``bindings`` option expects a map with two keys that refer to the controller to be protected:
+
+``class``
+
+    The fully qualified class name of the controller class.
+
+``method``
+
+    The name of the controller action method.
+
+``class``
+---------
+
+**type**: ``string``
+
+When the `type`_ option is set to ``entity``, the ``class`` option is used to decide whether or not
+the ACL has to be evaluated when checking if a user has access to a certain object. Each object ACL
+is only responsible for protecting access to a certain kind of objects.
+
+``group_name``
+--------------
+
+**type**: ``string``
+
+ACLs can optionally be grouped. A group is identified by its name.
```

Comment:
Talking to @DimaSoroka we decided to remove this options as it is not used. Please let me know if you don't agree with this change.